recipes/snakemake-executor-plugin-tes/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "snakemake-executor-plugin-tes" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/snakemake_executor_plugin_tes-{{ version }}.tar.gz
+  sha256: bbecd5c1e63a39e2ea84b752e8e4d68e64b16df686062f22e347cf97c7cd47b3
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  host:
+    - python >=3.11,<4.0
```

Comment:
With `python` not following semantic versioning, is this the correct version number limitation? Or will this anyways be handled properly by the `run_exports`?

Reply:
Good point. This is handled by the pinning, but we should probably completely remove the version constraint here.

Reply:
and in run.

Reply:
No wait, it is noarch. Hence we should keep something here. I am unsure. It is really a pity that Python does not follow semantic versioning.

Reply:
changed to 3.13, let us rather be conservative.